src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -153,4 +155,27 @@ abstract class IntegrationTestBase {
       .isOk
       .returnResult(CiagInductionResponse::class.java)
       .responseBody.blockFirst()!!
+
+  fun createGoal(
```

Comment:
I moved this method here into the base class `IntegrationTestBase` from `UpdateGoalTest`, and also added method args for the username that the JWT token should be created with (and therefore that is the user that is recorded as the createdBy etc fields on the created Goal)